onnxruntime/core/providers/cpu/tensor/pad.cc

```diff
@@ -517,22 +517,15 @@ Status Pad::Compute(OpKernelContext* ctx) const {
     slices_to_use = &slices_;
   }
 
-  Status pad_status{};
   switch (element_size) {
```

Comment:
nit: it's easier to set a break point if there's just one return point, but I guess you could set that in PadImpl

Reply:
ok, i can revert this in a future PR